tokenserver/assignment/__init__.py

```diff
@@ -4,6 +4,11 @@
 class INodeAssignment(Interface):
     """Interface definition for backend node-assignment db."""
 
+    def allocate_to_spanner(self, email):
```

Comment:
extremely picky nit: this method implements a predicate but is named like a verb, and behaves very differently to the similarly-named `allocate_user` method. Would you consider something like `should_allocate_to_spanner` instead?

I also don't think it needs to be part of the public API definition, but don't feel strongly if you want to keep it here.